charts/karpenter/values.yaml

```diff
@@ -103,7 +103,20 @@ controller:
     # -- SHA256 digest of the controller image.
     digest: sha256:bac5ea470c09df21eb3742cf9448e9b806138ed5b6321d4e04697bbdf122eac6
   # -- SecurityContext for the controller container.
-  securityContext: {}
+  securityContext:
```

Comment:
Should we just set all the defaults for this at the `podSecurityContext` level?

Reply:
Hi @jonathan-innis, Since allow privilege escalation, capabilities ReadOnly filesystem are available only for container security context, I modified all these under container security context so that it would be ease for someone to change. Let me your thoughts on this?

Reply:
Ah, got it. I didn't realize that some of the parameters were not available at the pod-level. Yep, that makes sense.

Reply:
Can we also set `runAsGroup` and `runAsUser` values at the pod-security context level on top of the other parameters?

Reply:
I have added the runAsGroup and runAsUser now.